### PR TITLE
feat(azure): add production azure config

### PR DESCRIPTION
**Pull request by olros**
## Proposed changes
-  Creates automatic deployment to production of Azure on push to master

Issue number: closes N/A

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [ ] Build (`make PR`) was run locally without failures

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Zenjjim** [comment]
Kan du legge inn at den automatisk migrerer databasen også?

**Review by Zenjjim — APPROVED** [review]

File: .github/workflows/deploy_prod.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        context: ./
+        file: ./Dockerfile
+        builder: ${{ steps.buildx.outputs.name }}
+        push: true
+        tags: leptonregistry.azurecr.io/lepton:latest
+        cache-from: type=local,src=/tmp/.buildx-cache
+        cache-to: type=local,dest=/tmp/.buildx-cache
```
Comment (norgor) [inline .github/workflows/deploy_prod.yml:44]: Does Azure perform periodic checks for whether latest tag has been updated?

Comment (olros) [reply .github/workflows/deploy_prod.yml:44]: Har satt opp continous deployment fra registry til app service slik at den blir oppdatert nå vi pusher til :latest

Comment (norgor) [reply .github/workflows/deploy_prod.yml:44]: Nice ❤

**Pull request merged**